apiserver/service/bitrise_config.go

```diff
@@ -37,6 +59,12 @@ func GetBitriseYMLHandler(w http.ResponseWriter, r *http.Request) {
 
 // PostBitriseYMLHandler ...
 func PostBitriseYMLHandler(w http.ResponseWriter, r *http.Request) {
+	contStr, err := fileutil.ReadStringFromFile(config.BitriseYMLPath)
```

Comment:
We're swallowing an error here, is that intentional? Is there a valid scenario when the file cannot be read (e.g. it's not there)? Same question for the other handler.